components/mpas-ocean/src/analysis_members/Registry_zonal_mean.xml

```diff
@@ -50,10 +50,10 @@
 		<var name="binBoundaryZonalMean" type="real" dimensions="nZonalMeanBinsP1" units="varies"
```

Comment:
@czender, this variable sometimes gets used with units of `radians` and sometimes with `m`.  Is there a way to handle this in a CF-compliant way?

Reply:
Just noting this is the only review comment I haven't addressed